### PR TITLE
chore(cypress): Favor firefox in cypress

### PR DESCRIPTION
**Pull request by mattwr18**
### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,8 +18,8 @@
     "cypress:backend": "cd backend && yarn run dev",
     "cypress:webapp": "cd webapp && yarn run dev",
     "cypress:setup": "run-p cypress:backend cypress:webapp",
-    "cypress:run": "cross-env cypress run",
-    "cypress:open": "cross-env cypress open",
+    "cypress:run": "cross-env cypress run --browser firefox",
+    "cypress:open": "cross-env cypress open --browser firefox",
```
Comment (roschaefer) [inline package.json]: That has the drawback that we cannot record our cypress tests anymore :cry: can we close?

File: cypress/integration/common/report.js [hunk 2]
```diff
@@ -169,7 +169,7 @@ Then('each list item links to the post page', () => {
 Then('I can visit the post page', () => {
   cy.contains(annoyingUserWhoMutedModeratorTitle).click()
   cy.location('pathname').should('contain', '/post')
-    .get('h3').should('contain', annoyingUserWhoMutedModeratorTitle)
+    .get('.ds-card-content .hyphenate-text').should('contain', annoyingUserWhoMutedModeratorTitle)
```
Comment (roschaefer) [inline cypress/integration/common/report.js]: Code creep?

File: package.json (hunk 1 above)
Comment (mattwr18) [reply package.json]: No, actually it doesn't, if you look at the build, or click on cypress, you can see that all the videos were recorded

File: cypress/integration/common/report.js (hunk 2 above)
Comment (mattwr18) [reply cypress/integration/common/report.js]: Could be, yes.
It failed locally, but that was in headless. Also, I was reading cypress docs and they suggest to never use tags as selectors.
We could open another PR to clean up our cypress code though

File: package.json (hunk 1 above)
Comment (mattwr18) [reply package.json]: I wouldn't close, is there any justifiable reason to use a different browser than what we recommend our users to use for e2e tests?

File: cypress/integration/common/report.js [hunk 3]
```diff
@@ -169,7 +169,7 @@ Then('each list item links to the post page', () => {
 Then('I can visit the post page', () => {
   cy.contains(annoyingUserWhoMutedModeratorTitle).click()
   cy.location('pathname').should('contain', '/post')
-    .get('title').should('contain', annoyingUserWhoMutedModeratorTitle)
+    .get('.base-card .title').should('contain', annoyingUserWhoMutedModeratorTitle)
```
Comment (mattwr18) [inline cypress/integration/common/report.js]: not related to this `PR` directly, but not even sure how this was passing!? the `title` doesn't have a `.` indicating it's a class :woman_shrugging:

File: cypress.json [hunk 4]
```diff
@@ -1,5 +1,8 @@
 {
   "projectId": "qa7fe2",
   "ignoreTestFiles": "*.js",
-  "baseUrl": "http://localhost:3000"
+  "baseUrl": "http://localhost:3000",
+  "env": {
+    "RETRIES": 2
+  }
```
Comment (mattwr18) [inline cypress.json]: Also, not related to `firefox` as a test runner, but unless I'm missing something, does anyone think `retries` are actually occurring? hopefully, we can see some evidence that this works

**Pull request merged**